### PR TITLE
Add find person by txn

### PR DESCRIPTION
**Pull request by WesleyBLDC**
1. Wrote the findtxn command, parser and etc with basic functionality
2. Wrote the unit tests for the command and parser
3. Checkstyle improved

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public class AddressBook implements ReadOnlyAddressBook {
 
     /*
      * The 'unusual' code block below is a non-static initialization block, sometimes used to avoid duplication
-     * between constructors. See https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
+     * between constructors. See https://docs.oracle.com/javase/tuxtorial/java/javaOO/initial.html
```
Comment (ajjajjajjajj) [inline src/main/java/seedu/address/model/AddressBook.java:27]: Minor spelling error here, fix in next merge.

File: src/main/java/seedu/address/logic/parser/FindTxnByPersonCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.FindTxnByPersonCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.OnlyOnePersonPredicate;
+import seedu.address.model.person.TxnContainsPersonPredicate;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
```
Comment (evanpy) [inline src/main/java/seedu/address/logic/parser/FindTxnByPersonCommandParser.java:11]: `FindCommand` should be `FindTxnByPersonCommand`

File: src/test/java/seedu/address/logic/commands/FindTxnByPersonCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.OnlyOnePersonPredicate;
+import seedu.address.model.person.TxnContainsPersonPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for
+ * {@codeFindTxnByPersonCommand}.
```
Comment (evanpy) [inline src/test/java/seedu/address/logic/commands/FindTxnByPersonCommandTest.java:20]: Might have missed a spacing `{@codeFindTxnByPersonCommand}`

**Review by evanpy — COMMENTED** [review]
Minor fixes. LGTM 👍🏻

**Pull request merged**